src/Facebook/FacebookClient.php

```diff
@@ -162,7 +162,11 @@ public function getBaseGraphUrl($postToVideoUrl = false)
      */
     public function prepareRequestMessage(FacebookRequest $request)
     {
-        $postToVideoUrl = $request->containsVideoUploads();
+        $pathElements = explode('/', $request->getUrl());
```

Comment:
In the start phase 
  -X POST \
  "https://graph-video.facebook.com/v2.3/1533641336884006/videos"  \
  -F "access_token=XXXXXXXXX" \
  -F "upload_phase=start" \
  -F "file_size=152043520"
there is no video file attached. so it's not safe to determine whether it is video request only by containsVideoUploads.